base/card-api.gts

```diff
@@ -162,7 +162,7 @@ export async function searchDoc<CardT extends CardConstructor>(model: InstanceTy
   for (let fieldName of Object.keys(getFields(model))) {
     // TODO we'll want to use a "queryValue" card field feature here
     let value = serializedGet(model, fieldName);
-    if (value) {
+    if (value != null) {
```

Comment:
I think we can eliminate this check entirely and just always put the value in the searchDoc, even when it's undefined.